src/Containers/Status.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { memo } from 'react';
+import styled from 'styled-components';
+import { transition, flex, position } from '../Utils/Mixins';
+import { SmallText, SmallTextProps } from '../Text';
+const STATUS_COLORS = {
+    prepared: '#28af00',
```

Comment:
These Colors are all magic numbers to me and for some reason not in the Theme Object meaning no theme support